src/services/auth.ts

```diff
@@ -92,7 +114,13 @@ export default function useAuth() {
 
   const addNewUser = useCallback(async (email, pwd) => {
     try {
-      await firebase.auth().createUserWithEmailAndPassword(email, pwd)
+      setUserCreationState({ status: 'loading' })
+
+      const { user } = await firebase.auth().createUserWithEmailAndPassword(email, pwd)
+
+      await axios.post(`${process.env.AUTH_API}/auth/setCustomClaims`, { uid: user?.uid })
```

Comment:
I just understood the process that you wanted to do, it is nice, you avoid the process of renewing the JWT that was in the tutorial that I shared with you. :+1: